packages/blaze/attrs.js

```diff
@@ -352,18 +358,18 @@ ElementAttributesUpdater.prototype.update = function(newAttrs) {
   const elem = this.elem;
   const handlers = this.handlers;
 
+  // Cache for last set values to avoid redundant DOM updates
+  if (!this._lastValues) this._lastValues = {};
+  const lastValues = this._lastValues;
+
   Object.getOwnPropertyNames(handlers).forEach((k) => {
     if (!has(newAttrs, k)) {
-      // remove attributes (and handlers) for attribute names
-      // that don't exist as keys of `newAttrs` and so won't
-      // be visited when traversing it.  (Attributes that
-      // exist in the `newAttrs` object but are `null`
-      // are handled later.)
       const handler = handlers[k];
```

Comment:
Instead of just removing he comment it would be great to have an updated comment that explains what exactly happens here (and more important, why it happens)